java/src/org/certificatetransparency/ctlog/MerkleAuditProof.java

```diff
@@ -0,0 +1,20 @@
+package org.certificatetransparency.ctlog;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class MerkleAuditProof {
+  public Ct.Version version;
+  public long treeSize;
+  public long leafIndex;
+  public List<byte[]> pathNode;
+
+  public MerkleAuditProof(Ct.Version version, long treeSize, long leafIndex) {
+    this.version = version;
+    this.treeSize = treeSize;
+    this.leafIndex = leafIndex;
+    pathNode = new ArrayList<byte[]>();
```

Comment:
Is that a reasonable implementation of `List<>` to use? I just looked around and poked in the dark until it compiled and the tests passed, really... :wink: